pom.xml

```diff
@@ -49,17 +53,44 @@
     <module>quickstart</module>
     <module>documentation</module>
   </modules>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.rest-assured</groupId>
+        <artifactId>rest-assured</artifactId>
+        <!-- Version 4.3.0 has binary incompatibilities -->
+        <version>4.2.0</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>cassandra</artifactId>
+        <version>1.12.4</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
```

Comment:
Trying to follow the convention of declaring versions inside management sections, but leaving configuration details to actual dependency/plugin declarations.